tests/unit/test_runner.py

```diff
@@ -100,7 +100,7 @@ def test_create(
 
     if runner.config.proxies:
         instance = instances[0]
-        env_proxy = instance.files.read_file("/opt/github-runner/.env")
+        env_proxy = instance.files.read_file("/home/ubuntu/github-runner/.env")
```

Comment:
Since the github runner path has to be changed at 2 places, shouldn't we make a constant or reuse the Path attribute if available ?